iconsdk/wallet/wallet.py

```diff
@@ -142,27 +141,59 @@ def store(self, file_path: PathLikeObject, password: str):
         except IsADirectoryError:
             raise KeyStoreException("Directory is invalid.")
 
-    def get_private_key(self) -> str:
+    def to_json(self, password: str) -> Dict[str, str]:
+        ret: Dict[str, str] = create_keyfile_json(
+            self.private_key,
+            bytes(password, 'utf-8'),
+            iterations=16384,
+            kdf="scrypt"
+        )
+        ret['address'] = self.get_address()
+        ret['coinType'] = 'icx'
+        return ret
```

Comment:
also `load_from_json(dict?)` needed.

Reply:
from_dict() and to_dict() are added to support your request.